### PR TITLE
Update ideogram tests.

### PR DESCRIPTION
**Pull request by mkcor**
Adresses #260. 

## About
Certain integration tests count (expected) elements by having Selenium find them (CSS selector). Sometimes, the underlying function times out before updated elements could be found and, hence, counted. 

## Description of changes
Delving into the Ideogram tests, I have standardized the finding / counting of elements. I have pushed frequently to have 'statistics' on CI test failures, and lately they haven't failed... So I'm thinking of not changing 
```
-    chromosomes = wait_for_elements_by_css_selector(driver, '.chromosome')
+    chromosomes = wait_for_elements_by_css_selector(driver, '.chromosome', timeout=20.0)
```
(at least, not yet).
## Before merging
- [ ] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [ ] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [ ] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

**Comment by mkcor** [comment]
PS: Letting the `plotly` dependency be `>=3.5.0` (in practice, I'm now running the latest version `3.7.1`) doesn't seem to change anything. I have explored the demo apps locally. There's still this error in the console when accessing Dash Alignment Viewer:
```
TypeError: the JSON object must be str, bytes or bytearray, not NoneType
```

File: src/lib/components/Ideogram.react.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {omit} from 'ramda';
  * datasets for organisms such as human, mouse, and any other
  * eukaryote. The Ideogram component can be used to compare
  * homologous features between chromosomes, and depict
- * haploid, diploid, aneuploidy genomes. It can also display
+ * haploid, diploid, aneuploid genomes. It can also display
```
Comment (shammamah) [inline src/lib/components/Ideogram.react.js]: Words like these make me remember why I switched my major from biochem 🤣

File: tests/test_ideogram.py [hunk 2]
```diff
@@ -227,16 +226,16 @@ def assert_callback(prop_value, nclicks, input_value):
     driver = dash_threaded.driver
 
     # assert 22 chromosomes + X and Y chromosomes
-    num_chromosoms = len(wait_for_elements_by_css_selector(driver, '.chromosome'))
-    assert num_chromosoms == 24
+    chromosomes = wait_for_elements_by_css_selector(driver, '.chromosome')
```
Comment (shammamah) [inline tests/test_ideogram.py]: Is there a particular reason that you changed these? (I'm just wondering if it somehow speeds things up, or ensures that we have a result before trying to test the assertion)

File: tests/test_ideogram.py [hunk 3]
```diff
@@ -629,7 +636,7 @@ def assert_callback(prop_value, nclicks, input_value):
 
     # assert the presence of homology region
     regions = wait_for_elements_by_css_selector(driver, '.syntenicRegion')
-    assert len(regions) != 0
+    assert len(regions) > 0
```
Comment (shammamah) [inline tests/test_ideogram.py]: Why was this changed? I guess it makes sense, since the length of a list cannot be negative, but I'm not sure this change affects performance or functionality. (Correct me if I'm wrong!)

File: tests/test_ideogram.py (hunk 2 above)
Comment (Bachibouzouk) [reply tests/test_ideogram.py]: I think it is to standardize the tests?

Comment (mkcor) [reply tests/test_ideogram.py]: For both reasons: standardize and get elements before taking their length (for possible debugging too).

File: tests/test_ideogram.py (hunk 3 above)
Comment (mkcor) [reply tests/test_ideogram.py]: It shouldn't affect performance or functionality, but if ever a length was negative, I would want to catch it!! Well, I'm for writing the meaningful version directly (we expect this length to be positive, so let's write it, it's easier on the reader too).
